examples/pub-sub-message-encryption/javascript/README.md

```diff
@@ -0,0 +1,39 @@
+# Sending and receiving messages with encryption
+
+This folder contains the code for encryption of messages (Typescript) - a demo of how you can encrypt and decrypt messages with [Ably](https://ably.com/docs/channels/options/encryption).
+
+## Getting started
+
+1. Clone the [Ably docs](https://github.com/ably/docs) repository where this example can be found:
+
+```sh
+git clone git@github.com:ably/docs.git
+```
+
+2. Change directory:
+
+```sh
+cd /examples/pub-sub-message-encryption/javascript/
+```
+
+3. Rename the environment file:
+
+```sh
+mv .env.example .env.local
```

Comment:
I think it's better to copy `.env.example` instead of renaming, this way we'll keep git index untouched